packages/k2-alpine/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropsWithChildren } from 'react'
+import { SxProp } from 'dripsy'
+import { View } from '../Primitives'
+import { useTheme } from '../..'
+
+export const Card = ({
+  sx,
+  children,
+  shouldCenterAlign = false,
+  backgroundColorOverride
```

Comment:
I don't think we should expose this background color prop. especially when it has to do with light and dark theme. that should be an internal implementation detail only.

Reply:
I also see some card with different color, let me reach out to design about it too.